### DISCUSSION TITLE
如何在构建消息时增加频道索引

### DISCUSSION THREAD

**Discussion by BlindspotCheat** [simple-robot/Discussions#173] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

kook

### 疑问描述

如题

**Comment by ForteScarlet** [comment]
尝试在构建 `At` 消息时，指定其类型为 [`KookMessages.AT_TYPE_CHANNEL`](https://docs.simbot.forte.love/components/kook/simbot-component-kook-core/love.forte.simbot.component.kook.message/-kook-messages/-a-t_-t-y-p-e_-c-h-a-n-n-e-l.html)，或直接使用 [`KookMessages.atChannel(...)`](https://docs.simbot.forte.love/components/kook/simbot-component-kook-core/love.forte.simbot.component.kook.message/-kook-messages/at-channel.html) 构建一个 `At` 消息
